### ISSUE TITLE
Accessibility page updates

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1222]
Recommendations for /accessibility-commitment page:

- [x] Remove p.lead class from 'xyz is committed to access for all' and add a period at the end.
- [x] Edit: 'Accessibility is A process' (add the 'a')
- [x] Would add a 'Contact Us' section at the bottom of page and move 'Questions? Please do not hesitate to contact us. You can do so here.' there.
- [x] The 'Accessibility links' header in the sidebar is linking to Adobe accessibility. 
- [x] Would just say 'Section 508' in sidebar.
- [x] Would remove 'Information' text in sidebar links.
- [x] New text for contact us: 'Questions, comments, concerns? <a href="contact" title="Contact us">Contact us.</a>' (note 'here' is not good accessibility practice as I've just learned - use descriptive text: https://webaccess.berkeley.edu/ask-pecan/click-here)
- [x] Delete 'n' at end of 'Chrome Accessibilityn' link in sidebar.
- [x] Update using code here: https://github.com/proudcity/wp-proudcity/issues/1222#issuecomment-367751155
- [x] Suggest defaulting URL to /accessibility or /accessibility-statement (or -policy) whatever might be the standard.

@aschmoe let me know if there's a file I can edit here and issue a PR. That might make it easier.

**Comment by lukefretwell** [comment]
Added to list above, but also remove p.lead class on the page.

Ex: https://elgl.proudcity.com/accessibility-commitment/

<img width="1229" alt="screen shot 2018-02-21 at 9 18 18 pm" src="https://user-images.githubusercontent.com/1569463/36521687-ee30ccc6-174c-11e8-9f3b-fc8d60e0974e.png">

**Comment by lukefretwell** [comment]
@aschmoe I found the file and edited it more, so it's ready to go. Note, I removed the sidebar and made it a full-width page. Updated checklist with last two items.

```
<div class="row">
	<div class="col-md-12 entry-content">
		<p><?php echo $entity_name; ?> is committed to access for all.</p>
		<p>Guided by Web Content Accessibility Guidelines (WCAG) standards, our objective is to be:</p>
		<ol>
			<li>Fully compliant with Section 508 and WCAG Level AA.</li>
			<li>Open and transparent about our process.</li>
			<li>Attentive to community feedback surrounding usability of our website.</li>
		</ol>
		<p>Accessibility is an ongoing process, and here are steps we have taken:</p>
		<ul>
			<li>Accessible forms: "aria" attributes, descriptive labels, tab-able interfaces</li>
			<li>Semantic markup and additional "aria" descriptors on UI components</li>
			<li>Tab-able navigational elements</li>
			<li>"Skiplinks" to important page elements and website sitemap</li>
			<li>Required "alt" descriptions on images</li>
			<li>Font size adjustment</li>
		</ul>
		<h3>PDF documents</h3>
		<p>Some of the documents that appear on this website are in PDF format. Those using assistive technology may be unable to read documents directly, unless you have the requisite plug-ins:</p>
		<ul>
			<li><a href="https://get.adobe.com/reader/" target="_blank">Download Acrobat Reader</a></li>
			<li><a href="http://www.adobe.com/accessibility/" target="_blank">Access Adobe Accessibility</a></li>
		</ul>
		<h3>Accessibility links</h4>
		<ul>
			<li><a href="https://www.w3.org/TR/WCAG/" target="_blank">WCAG Guidlines</a></li>
			<li><a href="https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-section-508-standards" target="_blank">Section 508</a></li>
		</ul>
		<h3>Browser tools</h4>
		<ul>
			<li><a href="http://www.microsoft.com/enable/products/default.aspx" target="_blank">Internet Explorer Accessibility</a></li>
			<li><a href="http://www.accessfirefox.com/" target="_blank">Firefox Accessibility</a></li>
			<li><a href="https://www.google.com/accessibility/all-products-features.html" target="_blank">Chrome Accessibility</a></li>
		</ul>
	    <h3>Contact us</h3>
	    <p>Please do not hesitate to <a href="/contact">contact us</a> us with feedback on our accessibility practices.</p>
	</div>
</div>
```
